.gitignore

```diff
@@ -7,6 +7,7 @@ target
 *.iws
 *.ipr
 .idea
+*DS_Store
```

Comment:
Add a dot in front of a file extension

Reply:
Fixed in https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/commit/d14dfd5420cd70c2bb864885bcb9ab336dca6ac0